MD/md-30/README.md

```diff
@@ -0,0 +1,175 @@
+- **Rooms**: Digital spaces associated with a registered Name, accessible by the Name's owner or key holders.
+- **Keys**: Access key to Name's rooms. Mintable and burnable according to a bonding curve.
+- **Bonding Curve**: A mathematical model used to determine minting and burning prices for access keys.
+
+## Desiderata
+
+### D1: Decentralized and Permissionless Name System
+
+**User Journey**: Users can register human-readable names on the blockchain and access related services.
+
+**Justification**: A decentralized name system is essential for empowering users with on-chain identities and ensuring they can interact with the blockchain without technical friction.
+
+**Guidance and suggestions**:
+
+- Names should be fully decentralized and governed by the community, adhering to the principles of openness and transparency.
+
+### D2: Bonding Curve for Key Minting and Burning
+
+**User Journey**: Users or services can mint or burn access keys based on a bonding curve.
+
+**Justification**: A bonding curve ensures that access keys' prices are dynamically adjusted, preventing speculation and enabling fair access to Names and Rooms.
+
+### D3: Community-Driven Revenue Models
+
+**User Journey**: The community can implement their own revenue models via Rooms and Keys. Movement drives the first implementation with Ductus, a backend serving the name service and rooms, and Domus, the frontend that allows minting, trading, viewing of names and keys, and using chat rooms.
+
+**Justification**: Allowing the community to drive revenue through their traffic contributions promotes incentivized participation, enabling them to earn protocol-generated funds.
+
+**Guidance and suggestions**:
+
+- For Rooms and Keys, community members should be able to pass their address as `application_fee_recipient` to receive funds generated by the traffic they drive to the protocol.
+
+### D4: Expandable Name and Room Ecosystem
+
+**User Journey**: Developers can build custom applications and infrastructure using the Name and Room system.
+
+**Justification**: A flexible framework encourages innovation, enabling third-party developers to create services, tools, or applications that enhance the user experience within the MNS ecosystem.
+
+**Guidance and suggestions**:
+
+- Name and Room ownership should be tied to flexible access key resources, allowing for extensibility through third-party integrations and custom services.
+
+### D5: Each Movement Name should grant access to its own Domain for .move TLD
+
+**User Journey**: Users acquire a `.move` domain. This allows the user to host an application running on http/https.
+
+**Justification**: Domains are yet to be fully implemented for any web3 name service and `.move` is an elegant TLD with intrinsic value for a global market.
+
+## Design
+
+### Protocol Units
+
+- **NameRegistry**: Manages the registration and ownership of Names on the Movement blockchain, similar to the Aptos Name Service.
```

Comment:
This starts to get more towards the actual proposal or recommendations herein. I would move this up to the top under a **Terms** section.

Reply:
You can also put all of this down in the appendix.